container-chains/templates/frontier/runtime/src/lib.rs

```diff
@@ -652,7 +637,7 @@ impl pallet_dynamic_fee::Config for Runtime {
 }
 
 parameter_types! {
-    pub DefaultBaseFeePerGas: U256 = U256::from(1_000_000_000);
+    pub DefaultBaseFeePerGas: U256 = U256::from(2_000_000_000);
```

Comment:
This internal gets multiplied by the ideal target fullness (50%) to calculate the minimum gas price below which we cannot drop